docs/podman-ps.1.md

```diff
@@ -103,6 +103,13 @@ Valid filters are listed below:
 
 Print usage statement
 
+**--sync**
+
+Force a sync of container state with the OCI runtime.
+In some cases, a container's state in the runtime can become out of sync with Podman's state.
+This will update Podman's state based on what the OCI runtime reports.
+Forcibly syncing is much slower, but can resolve inconsistent state issues.
+
```

Comment:
IDK what others think, but I don't think --sync belongs in a command that's used to print out information on the container.  Since we've established a `podman container exists` command, I think it would be better to have a `podman container sync` command.

That's my $.02 anywho.

Reply:
I think it's useful when attached to `ps` and `rm` because you might want to be absolutely certain you're getting accurate container information for both, but I agree a dedicated command would also be welcome.

Reply:
If not already created, could you make an issue or jira card for the dedicated command please?

Reply:
Per @rhatdan, we're going to try and handle this automatically now - we'll throw errors when a runtime-related issue occurs, and retry with a sync. Since we're internally detecting and handling issues, we probably don't need a dedicated command (the sync flag on `ps` still kind of makes sense, there are no errors to handle since `ps` no longer hits runc). I'll make a card for that work.